scripts/release/README.md

```diff
@@ -0,0 +1,19 @@
+Notes for release managers
+---
+
+This document describes how to make a GA4GH schemas release.
+
+Setup your environment
+1. Copy (or incorporate) the settings.xml file to ```~/.m2/settings.xml```
+2. Create a GPG key, and add it to KEYS
```

Comment:
May want to link to http://central.sonatype.org/pages/working-with-pgp-signatures.html, since there is a least one gotcha (Maven as well as Nexus Repository Manager can only verify against a primary key, and most implementations of gpg create a sub key for signing by default, which will need to be deleted; see the section Delete a Sub Key) and a few additional steps (e.g.  distribute your public key to a key server).